Toolchain/default.nix

```diff
@@ -21,9 +19,6 @@ stdenv.mkDerivation {
     rsync
     texinfo
     unzip
-  ];
-
-  buildInputs = [
     e2fsprogs
     fuse2fs
```

Comment:
there's duplicates here for e2fsprogs and fuse2fs

Reply:
Which one is needed? I'll edit the pr

Reply:
oh, I meant that both of those packages are duplicates with further up in the package list (they used to be both in nativeBuildInputs and buildInputs)